### PR TITLE
Create discussion.md

### PR DESCRIPTION
**Pull request by ryandemopoulos**
Adding another issue type for discussions (and new ideas that need more brainstorming).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/discussion.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+name: Discussion
+about: Start a discussion topic, or brainstorm an new idea, related to WinUI
```
Comment (jevansaks) [inline .github/ISSUE_TEMPLATE/discussion.md]:
```suggestion
about: Start a discussion topic, or brainstorm a new idea, related to WinUI
```

File: .github/ISSUE_TEMPLATE/discussion.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+---
+
+<!-- This is a template for discussion issues.
+These issues invite members of our community to discuss any topics or ideas related to WinUI.  For example, you might
+start a Discussion issue to see if others are interested in a cool new idea you've been thinking about that isn't quite
+ready to be filed as a more formal Feature Proposal.
+
+Discussion issues can also just be used to chat about non-feature-related topics, such as how WinUI relates to other
+things happening in our industry, or stories you'd like to share about how WinUI meets (or doesn't) meet your needs.
```
Comment (jevansaks) [inline .github/ISSUE_TEMPLATE/discussion.md]:
```suggestion
things happening in our industry, or stories you'd like to share about how WinUI meets (or doesn't meet) your needs.
```

File: .github/ISSUE_TEMPLATE/discussion.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+-->
+
+# Discussion: [your title here] 
+<!-- Add a title for your Discussion issue. Please be short and descriptive, and write body text in this section. -->
+
+## Related Links
+<!-- Please list any related links or referneces to this discussion issue -->
```
Comment (jesbis) [inline .github/ISSUE_TEMPLATE/discussion.md]:
```suggestion
<!-- Please list any related links or references to this discussion issue -->
```

File: .github/ISSUE_TEMPLATE/discussion.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+name: Discussion
+about: Start a discussion topic, or brainstorm an new idea, related to WinUI
+title: 'Discussion:'
```
Comment (jevansaks) [inline .github/ISSUE_TEMPLATE/discussion.md]:
```suggestion
title: 'Discussion: [your discussion title]'
```

File: .github/ISSUE_TEMPLATE/discussion.md (hunk 3 above)
Comment (jevansaks) [inline .github/ISSUE_TEMPLATE/discussion.md]:
```suggestion
<!-- Please list any related links or references to this discussion issue -->
```

File: .github/ISSUE_TEMPLATE/discussion.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+labels: discussion
+assignees: ''
+
+---
+
+<!-- This is a template for discussion issues.
+These issues invite members of our community to discuss any topics or ideas related to WinUI.  For example, you might
```
Comment (jesbis) [inline .github/ISSUE_TEMPLATE/discussion.md]:
```suggestion
Discussion issues can be about any topics or ideas related to WinUI.  For example, you might
```
First sentence sounds like it's explaining why we created it and not why/how to use it - maybe make it more opener-oriented?

**Comment by ryandemopoulos** [comment]
Thanks @jevansaks and @jesbis -- great catches on the editing.  I accepted all suggestions.

**Pull request merged**